src/views/CountryThreeWProjects/index.tsx

```diff
@@ -0,0 +1,495 @@
+
+        return [
+            projectsOngoing,
+            target,
+            ongoingBudget,
+            programmeTypes,
+            projectStatusTypes,
+            activeNS,
+        ];
+    }, [filteredProjectList]);
+
+    const districtGroupedProject = useMemo(() => {
+        const districtDenormalizedProjectList = denormalizeList(
+            ongoingProjects ?? [],
+            (project) => project.project_districts_detail,
+            (project, district) => ({
+                ...project,
+                project_district: district,
+            }),
+        );
+
+        return listToGroupList(
+            districtDenormalizedProjectList,
+            (d) => d.project_district.id,
+        );
+    }, [ongoingProjects]);
+
+    const [
+        localNSProjects,
+        otherNSProjects,
+    ] = useMemo(() => ([
+        ongoingProjects.filter((project) => project.reporting_ns === project.project_country),
+        ongoingProjects.filter((project) => project.reporting_ns !== project.project_country),
+    ]), [ongoingProjects]);
+
+    const tableColumns = useMemo(() => ([
+        createStringColumn<Project, number>(
+            'ns',
+            strings.threeWTableNS,
+            (item) => item.reporting_ns_detail?.society_name,
+        ),
+        createStringColumn<Project, number>(
+            'name',
+            strings.threeWTableProjectName,
+            (item) => item.name,
+        ),
+        createStringColumn<Project, number>(
+            'sector',
+            strings.threeWTableSector,
+            (item) => item.primary_sector_display,
+        ),
+        createNumberColumn<Project, number>(
+            'budget',
+            strings.threeWTableTotalBudget,
+            (item) => item.budget_amount,
+            undefined,
+        ),
+        createStringColumn<Project, number>(
+            'programmeType',
+            strings.threeWTableProgrammeType,
+            (item) => item.programme_type_display,
+        ),
+        createStringColumn<Project, number>(
+            'disasterType',
+            strings.threeWTableDisasterType,
+            (item) => item.dtype_detail?.name,
+        ),
+        createNumberColumn<Project, number>(
+            'peopleTargeted',
+            strings.threeWTablePeopleTargeted,
+            (item) => item.target_total,
+            undefined,
+        ),
+        createNumberColumn<Project, number>(
+            'peopleReached',
+            strings.threeWTablePeopleReached,
+            (item) => item.reached_total,
+            undefined,
+        ),
+        createActionColumn(
+            'actions',
+            (project: Project) => ({
+                children: (
+                    <ProjectActions
+                        onProjectDeletionSuccess={reTriggerProjectListRequest}
+                        className={styles.actions}
+                        project={project}
+                    />
+                ),
+            }),
+        ),
```

Comment:
We should not use createActionColumn to directly pass React components.
Let's create a custom createActionColumn for this